src/sleep/sleep.c

```diff
@@ -366,6 +366,8 @@ static int freeze_thaw_user_slice(const char **method) {
         if (r < 0)
                 return log_debug_errno(r, "Failed to open connection to systemd: %m");
 
+        sd_bus_set_method_call_timeout(bus, USEC_PER_SEC);
```

Comment:
Is 1 second really enough? IIUC, freezing either works or you get onto 25 second timeout, so maybe slightly bigger value to account for busy dbus-daemon or whatever might be better. Say 5 seconds or so.

Reply:
Bumped to 1.5 seconds.